resources/social-media.md

```diff
@@ -0,0 +1,29 @@
+# RustBridge Social Media Guide
+Social Media can be a very powerful tool to leverage your event, but also to motivate attendees to share their results! Because it’s so powerful, it’s very helpful and sometimes necessary to set concrete rules for usage by not only organizers and teachers, but also attendees.
+
+
+## Photo Guidelines
+
+This is the part that is necessary. You want to make sure that your event is governed by a photo taking policy, meaning that you want to ask for people’s *consent* before you take their picture. The same thing goes for attendees taking pictures.
+
+Assume that every individual person is **not** ok with having their picture taken until you ask them to. The easiest way to do this is to sort this out at the beginning by asking the crowd of attendees if they’re okay with it. If not everyone is, *don’t* call them out — tell them that it’s okay, and ask the rest of the group to either be careful with or refrain from taking pictures with people in them altogether.
+
+
+## Social Media Hashtags
+
+A cool thing to do is to encourage attendees and teachers to post their learnings or other findings under a hashtag on platforms like Twitter or Instagram. This hashtag should be called `#RustBridgeLOCATION` or `#RustBridgeIDENTIFIER`, so like `#RustBridgeBerlin` or `#RustBridgeDC`. It’s also okay to just post under the `#RustBridge` hashtag, or both! Incentivize tagging posts by offering to retweet them with your event’s account, or the official RustBridge account, but be careful that they don’t violate your photo taking rules, as discussed in the previous section.
```

Comment:
you might mention that the students can DM their posts to the appropriate accounts to help facilitate RTing